noir/noir-repo/compiler/noirc_evaluator/src/brillig/brillig_ir.rs

```diff
@@ -217,7 +216,10 @@ impl BrilligContext {
             array_ptr,
             index,
             index_of_element_in_memory,
-            BrilligBinaryOp::Field { op: BinaryFieldOp::Add },
+            BrilligBinaryOp::Integer {
```

Comment:
we can just use `self.memory_op` instead of binary_instruction :D

Reply:
Awesome thanks.